demos/twa-webview-fallback/src/main/res/values/strings.xml

```diff
@@ -19,12 +19,25 @@
         <item>https://pm5-monitor-c63a2.firebaseapp.com/</item>
         <item>https://jakearchibald.github.io/</item>
     </string-array>
+
     <string name="asset_statements">
         [{
             \"relation\": [\"delegate_permission/common.handle_all_urls\"],
             \"target\": {
                 \"namespace\": \"web\",
                 \"site\": \"https://pwa-directory.appspot.com/\"}
+        }],
+        [{
+            \"relation\": [\"delegate_permission/common.handle_all_urls\"],
+            \"target\": {
+                \"namespace\": \"web\",
+                \"site\": \"https://pm5-monitor-c63a2.firebaseapp.com/\"}
```

Comment:
What's this about?

Reply:
The second is a PWA inside PWA Directory to make testing navigation from one domain to the other easy.